priv/eleveldb_multi.schema

```diff
@@ -116,8 +116,8 @@
 {mapping,
   "multi_backend.$name.leveldb.compression",
   "riak_kv.multi_backend", [
-  {default, on},
-  {datatype, flag},
+  {default, snappy},
+  {datatype, [{atom, on}, {atom, off}, {atom, snappy}, {atom, lz4}]},
```

Comment:
There is a minor b/w compatibility issue here we should think about it.  The previous type was flag, so 'true' and 'false' are allowable, and maybe even others ('0', and '1'?   Not sure, I am not a cuttlefish expert).  So if someone used 'true' before, then they will have a start failure on an upgrade.